feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.html

```diff
@@ -12,7 +12,12 @@
         <span> {{ product.name }} </span>
       </ng-container>
     </div>
-    <a role="link" (click)="triggerDetails()" tabindex="0">
+    <a
+      role="link"
+      (click)="triggerDetails()"
+      (keyup)="clickOnEnter($event)"
+      tabindex="0"
+    >
```

Comment:
This is semantically the button. Why don't we use `<button>` instead?

We shouldn't add unnecessary complexity (methods) to TS class just for handling custom UI event, when it's not justified.

Using `<button>` would support Enter by natively.